add sl docs (#3392)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/build-semantic-layer-intro.md

---
title: "Build your Semantic Layer"

> id: build-semantic-layer-intro

Reviewer: Can we trim the length here? For consistency, can we come to an agreement between using `sl` and `semantic-layer` in the id? We flip-flop between the two on different pages.

Reviewer 2: yep, good idea! will do

description: "Learn about MetricFlow and build your Semantic Layer with Semantic models and Metrics"
sidebar_label: Build your Semantic Layer

> tags: [Metrics, Semantic Layer]

Reviewer: ```suggestion
tags: [Metrics, Semantic Layer, Governance]
```

hide_table_of_contents: true
---

> Build your semantic layer seamlessly using the dbt Semantic Layer, powered by MetricFlow. Use constructs like semantic models and metrics to avoid repetition and optimize your development workflow.

Reviewer: ```suggestion
Build your semantic layer seamlessly using the dbt Semantic Layer, powered by MetricFlow. Use constructs like [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview) to avoid repetition and optimize your development workflow.
```

> Seamlessly build your semantic layer using [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview), avoid repetition, and  optimize your workflow with concepts you're already familiar with using dbt metrics, powered by **MetricFlow**. 

Reviewer 3: This sentence reads a bit weird. If we're going to use a tricolon (1,2, & 3) can we hammer home the final point?

Reviewer 2: gotcha! made some changes to the text

> Seamlessly build your semantic layer using [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview), avoid repetition, and  optimize your workflow with concepts you're already familiar with using dbt metrics, powered by **MetricFlow**. 

Reviewer 2: ```suggestion
Seamlessly build your semantic layer using [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview), avoid repetition, and  optimize your workflow with concepts you're already familiar with using dbt metrics, powered by **MetricFlow**. 
```
```suggestion
Build your semantic layer seamlessly using dbt Metrics, powered by **MetricFlow**. Use [semantic models](/docs/build/semantic-models and [metrics](/docs/build/metrics-overview) to avoid repetition and optimize your workflow with concepts you're already familiar with. 
```

> Seamlessly build your semantic layer using [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview), avoid repetition, and  optimize your workflow with concepts you're already familiar with using dbt metrics, powered by **MetricFlow**. 

Reviewer 2: ```suggestion
Build your semantic layer seamlessly using dbt Metrics, powered by **MetricFlow**. Use [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview) to avoid repetition and optimize your workflow with concepts you're already familiar with.
```

These sections explain how you can use the dbt Semantic Layer to define your metrics and [semantic graph](/docs/build/metricflow-core-concepts#semantic-graph), as well as provide data governance over how company metrics are defined and managed. 

> A few things to note before you begin:

Reviewer: ```suggestion
Here are a few considerations to note before you begin:
```

> - Currently for the dbt Core v1.6 beta, you can can define metrics in the [command line (CLI)](/docs/core/about-the-cli) using the [dbt and MetricFlow spec](https://github.com/dbt-labs/dbt-core/discussions/7456), and query them using the CLI. 

Reviewer 3: We might want to correct this because I don't know if `define metrics in the command line` is the right way to describe defining them in yml and querying 🤔

Reviewer 2: updated per Callum's feedback

- You can define metrics in YAML and query them with the [command line (CLI)](/docs/core/about-the-cli) on dbt Core version 1.6 beta, using the [new Semantic Layer specifications](https://github.com/dbt-labs/dbt-core/discussions/7456).
  * Note: dbt Cloud support coming soon.

> - dbt Labs is working with [integration](https://www.getdbt.com/product/semantic-layer-integrations) partners to release a beta of integrations and we're working to introduce other consumption methods like Python and JDBC. <br /><br />

Reviewer: ```suggestion
- dbt Labs is working with [integration partners](https://www.getdbt.com/product/semantic-layer-integrations) to release a beta of integrations and we're working to introduce other consumption methods like Python and JDBC. <br /><br />
```

<div className="grid--4-col">

<Card

>     title="Quickstart"

Reviewer: Given the extensive work we've done to the Quickstarts (separating them out into their own section), would it be better to retitle this to something like "Getting Started"?

>     body="Learn how to create a semantic model, metric, and test and upload your metric using the Semantic Layer with MetricFlow."

Reviewer: Frequently, `powered by` is a careful branding decision, like the product name, so we may want to stick with that language for consistency. 
```suggestion
    body="Learn how to create a semantic model, a metric, and test and upload your metric using the Semantic Layer powered by MetricFlow."
```

    link="/docs/build/sl-getting-started"
    icon="rocket"/>

<Card
    title="About MetricFlow"
    body="Understand MetricFlow's core concepts, key principles, and how to use this powerful tool."
    link="/docs/build/metricflow-core-concepts"
    icon="rocket"/>

  <Card
    title="Semantic model"
    body="Use Semantic models as the basis for defining data. They act as nodes in the semantic graph, with entities connecting them."
    link="/docs/build/semantic-models"
    icon="rocket"/>

  <Card
    title="Metrics"

>     body="Metrics are functions that take in various parameters (such as measures, constraints) to define new quantitative indicators."

Reviewer: Will this definition be clear to outsiders looking in on these functions for the first time? It seems a little vague compared to the others.
